io.github.chcandido.brModelo.metainfo.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="desktop-application">
+  <id>io.github.chcandido.brModelo</id>
+  <launchable
+    type="desktop-id"
+  >io.github.chcandido.brModelo.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  <developer_name>Carlos Henrique Candido</developer_name>
+  <name>brModelo</name>
+  <summary
+  >Entity-relationship diagram tool for making conceptual and logical database models.</summary>
```

Comment:
```suggestion
  <summary>Entity-relationship diagram tool for making conceptual and logical database models.</summary>
```

Reply:
fixed in [`fabed18` (#2876)](https://github.com/flathub/flathub/pull/2876/commits/fabed18e1de91483e74f8d6b6930eaed25d22bab).